bubo/bot_commands.py

```diff
@@ -609,3 +612,15 @@ async def _users(self):
         if not text:
             text = help_strings.HELP_USERS
         await send_text_to_room(self.client, self.room.room_id, text)
+    async def _keys(self):
+        subcommand = self.args[0]
+        if subcommand == "create":
+            if self.args[1] == "permanent":
```

Comment:
Similarly, we may not have `self.args[1]` which would raise a `KeyError`. Use `if len(self.args) > 1` for example to guard.